src/components/EmailList/EmailList.jsx

```diff
@@ -0,0 +1,63 @@
+import EmailItem from "../EmailItem/EmailItem";
+import SearchBar from "../SearchBar/SearchBar";
+import EmailString from "../EmailString"
+import { useEffect, useState } from "react"
+
+function EmailList() {
+
+    const [emailString, setEmailString] = useState(false)
```

Comment:
`emailString` is not a good name for this state, because it is not a string, it is an array. Can you think of a better name?